camviewer_ui_impl.py

```diff
@@ -2711,16 +2711,16 @@ def getConfig(self):
         self.cfg = None
 
 
-def write_camera_config(self: GraphicUserInterface) -> None:
-    f = open(self.cfgdir + self.cameraBase, "w")
-    f.write("projsize    " + str(self.projsize) + "\n")
-    f.write("viewwidth   " + str(self.viewwidth) + "\n")
-    f.write("viewheight  " + str(self.viewheight) + "\n")
+def write_camera_config(gui: GraphicUserInterface) -> None:
```

Comment:
Just for my own understanding--why was self changed to gui?

Reply:
In `python` typically `self` is used for bound class methods (by convention, not by necessity). Since this isn't a bound class method, I decided to rename the variable so people won't get confused.